GitFyle.Core.Api/Brokers/Storages/StorageBroker.Contributions.References.cs

```diff
@@ -0,0 +1,30 @@
+﻿// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+using GitFyle.Core.Api.Models.Foundations.Contributions;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace GitFyle.Core.Api.Brokers.Storages
+{
+    internal partial class StorageBroker
+    {
+        void AddContributionConfigurations(EntityTypeBuilder<Contribution> builder)
+        {
+            builder.HasIndex(contribution => new
```

Comment:
Add an IsRequired for properties:
RepositoryId
ContributorId
ContributionTypeId